DuckDB.NET/DuckDBTimeOnly.cs

```diff
@@ -24,27 +24,37 @@ public DuckDBTimeOnly(byte hour, byte min, byte sec, int msec)
 
         public byte Sec { get; }
 
-        public int Msec { get; }
+        public int Microsecond { get; }
+
+        public long ToTicks() => Utils.GetTicks(Hour, Min, Sec, Microsecond);
```

Comment:
I think this should be property to align with `DateTime` and `TimeOnly`

Reply:
Done!